### PR TITLE
#8957: Add user and host data to sweeps results

### PR DESCRIPTION
**Pull request by jdesousa-TT**
### Ticket
#8957

### Problem description
It was not possible to tell who ran the sweep tests, and on what machine they were run on.

### What's changed
Adds user and host data to test results so we know which user/machine the test was run by/on.

### Checklist
- [x] Post commit CI passes
- [ ] Model regression CI testing passes (if applicable)
- [ ] New/Existing tests provide coverage for changes

### CODE REVIEW

File: tests/sweep_framework/runner.py [hunk 1]
```diff
@@ -27,6 +27,20 @@ def git_hash():
...
+        return subprocess.check_output(["uname", "-n"]).decode("ascii").strip()
+    except:
+        return "Couldn't get hostname!"
+
+
+def get_username():
+    try:
+        return os.environ["USER"]
+    except:
+        return "Couldn't get username!"
```
Comment (xanderchin) [inline tests/sweep_framework/runner.py:41]: I wouldn't try to recover from these exceptions. If you can't get the hostname or username, there must be some critical failure.

File: tests/sweep_framework/runner.py [hunk 2]
```diff
@@ -74,49 +88,47 @@ def execute_suite(test_module, test_vectors, pbar_manager, suite_name):
...
             test_vector.pop("invalid_reason")
             test_vector.pop("status")
             test_vector.pop("validity")
-        if p is None:
-            p = Process(target=run, args=(test_module, input_queue, output_queue))
-            p.start()
-        try:
-            if MEASURE_PERF:
-                # Run one time before capturing result to deal with compile-time slowdown of perf measurement
+            if p is None:
```
Comment (xanderchin) [inline tests/sweep_framework/runner.py:95]: Some of these changes seem unrelated to the PR?

**Review by xanderchin — CHANGES_REQUESTED** [review]

File: tests/sweep_framework/runner.py (hunk 2 above)
Comment (jdesousa-TT) [reply tests/sweep_framework/runner.py:95]: It's just to avoid duplicating the "host" and "user" assignments if there is an invalid vector, just moves the running to the else block of the invalid check and assigns the host and user after regardless of if the vector ran.

**Review by sjameelTT — APPROVED** [review]

**Review by xanderchin — APPROVED** [review]
